### ISSUE TITLE
Suggest accessibility: make announcements more concise

### ISSUE THREAD

**Issue by isidorn** [Microsoft/vscode#3861]
We got feedback from users for debug accessibility to make the announcements as concise as possible. Here are some ideas on how to make it more concise for suggest widget:
1. Do not announce 'suggestion' after each suggestion, instead of that announce when suggest widget opened and closed - that way the user will know that what he is hearing are suggestions inside the widget
2. instead of saying 'name, suggestion accepted' just say 'name, accepted'

@zersiax opinions very welcome

**Comment by zersiax** [comment]
Haha that user feedback came from me among others :) Yes, in this case
that is even more important. I think just 'list opened'  and 'list
closed'  or something similar should be enough, definitely not longer,
for announcing the intelliSense popping up.
As for what to announce for the suggestions, for that I would need to
know what is on the screen visually. For example, in Visual Studio
when methods are autocompleted using IntelliSense I think it gives the
short synopsis as well, at least for screenreaders. Are we interested
in doing that as well for Code? Or do we just report the current
autocompletion and give some kind of mechanism for the user to report
the help text using a hotkey, if the help text even exists in Code?

**Comment by alexandrudima** [comment]
@isidorn There is no need to announce the suggest widget opening, that would only add more noise. There is still a need to announce "suggestion" after each suggestion as now the entire line gets read again on cursor up/down. I will write down more details in #3787. I have shortened the message when accepting a suggestion, as per your point number 2.

@zersiax I will describe shortly the current state of the suggest widget in #3787
